src/aasthaJJun20/NumbersAndArrays.java

```diff
@@ -0,0 +1,77 @@
+package aasthaJJun20;
+
+public class NumbersAndArrays {
+	//This method is written just to display the array elements in output
+	void displayArray(int[] input) {
+		System.out.print("Input Array : { ");
+		for(int index = 0; index < input.length; index++) {
+			System.out.print(input[index]+" ");;
+		}
+		System.out.println("}");
+	}
+	
+	// Program 1: Find the missing number in integer array of 1 to 10?
+	void findMissingNumberFromArray(int[] input) {
+		displayArray(input);
+		int actualSum = 0, differedSum = 0;
+		for (int index = 1; index <= 10; index++) {
+			actualSum += index;
+		}
+		for (int index = 0; index < input.length; index++) {
+			differedSum += input[index];
+		}
+		System.out.println("The missing integer is: " + (actualSum - differedSum));
+
+	}
+
+	// Program 2: Find duplicate number on Integer array in Java? when exactly one number is repeated in the array
+	void findDuplicateInArray(int[] input) {
+		displayArray(input);
+		for (int index = 0; index < input.length; index++) {
+			for (int innerIndex = 1; innerIndex < input.length; innerIndex++) {
+				if (input[index] == input[innerIndex] && index != innerIndex) {
+					System.out.println("Duplicate number is: " + input[index]);
```

Comment:
This double for loop is working as expected. But it could be written in a little better way and with that the if() condition will also be much cleaner. PFB:-

for (int index = 0; index < input.length; index++) {
		for (int innerIndex = index+1; innerIndex < input.length; innerIndex++) {
				if (input[index] == input[innerIndex] ) {}
              }
}

Reply:
Right, this logic looks more simplified, will change it. Thanks